specs/lib/commentOnClosedIssueSpec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var fsExtra = require('fs-extra');
+var Promise = require('bluebird');
+var requestPromise = require('request-promise');
+
+var getUniqueMatch = require('../../lib/getUniqueMatch');
+
+var commentOnClosedIssue = require('../../lib/commentOnClosedIssue');
+
+describe('commentOnClosedIssue', function() {
+    it('throws if `url` or `message` is undefined', function () {
+        expect(function() {
+            commentOnClosedIssue();
```

Comment:
When testing for missing parameters, make sure all other parameters are actually passed in and also specifically pass in `undefined` for the missing parameter to improve readability.